java/src/main/java/ai/rapids/cudf/Cuda.java

```diff
@@ -21,13 +21,19 @@
 public class Cuda {
   // Defined in driver_types.h in cuda library.
   static final int CPU_DEVICE_ID = -1;
+  static final long CUDA_STREAM_LEGACY = 1;
+  static final long CUDA_STREAM_PER_THREAD = 2;
   private static final Logger log = LoggerFactory.getLogger(Cuda.class);
   private static Boolean isCompat = null;
 
   static {
     NativeDepsLoader.loadNativeDeps();
   }
 
+  private static long defaultStream() {
+    return isPtdsEnabled() ? CUDA_STREAM_PER_THREAD : CUDA_STREAM_LEGACY;
+  }
```

Comment:
This doesn't need to be a function, since it's never going to change value.  This should be computed once so we avoid the JNI call each time.
```suggestion
  private static long DEFAULT_STREAM_ID = isPtdsEnabled() ? CUDA_STREAM_PER_THREAD : CUDA_STREAM_LEGACY;
```

Reply:
Done.